lib/semantics/check-do.cc

```diff
@@ -318,8 +269,9 @@ class DoConcurrentLabelEnforce {
 
   void checkLabelUse(const parser::Label &labelUsed) {
     if (labels_.find(labelUsed) == labels_.end()) {
-      context_.Say(currentStatementSourcePosition_,
-          "control flow escapes from DO CONCURRENT"_err_en_US);
+      SayWithDo(context_, currentStatementSourcePosition_,
+          std::move("Control flow escapes from DO CONCURRENT"_err_en_US),
```

Comment:
I do not think you need to explicitly move this. It is not wring, but since you are constructing the message inside `SayWithDo` call, it is a temporary and it should already be something that can be moved/copy elided by C++ compilers. Have you tried without the `std::move` ?
(This comment applies to other `SayWithDo` calls)

Reply:
Thanks, Jean.  I removed the ```std::move()``, and things still seem to work.